### ISSUE TITLE
.html instead of .cshtml for partials

### ISSUE THREAD

**Issue by dehghani-mehdi** [RickStrahl/Westwind.RazorHosting#12]
Hi,

Many thanks for this tool, it's awesome.

Is there any way to use `.html` instead of `.cshtml` for partials? (_when I use `@RenderPartial("~/header.cshtml", Model)` everything works fine, but not for `@RenderPartial("~/header.html", Model)`, `RenderTemplate` returns `null`_)

I used your `AppTemplates` example, btw.

Mehdi

**Comment by RickStrahl** [comment]
My first question is - why does the name matter? It's a partial template, so why not just use .cshtml for the extension and you're good to go.

I'm not quite sure why that .html doesn't work though assuming the page has the same content and doesn't have an error on it. Check the error message to see what the problem is.

**Comment by dehghani-mehdi** [comment]
wow! really fast reply, thanks.
It's not really big deal, I know, but I have large system with many partials (.html files), I used my own template engine, but that was not good and powerful as Razor, so I decided to use Razor, I googled a lot, tired couple of tools, like RazorEngine, even read your article, finally found `Westwind.RazorHosting`, I found this tool good enough for me and my group. so will be nice to stay with `.html`, also I see your library works fine with `.html` as main template, but not for `partials`, so I think the extension should not be matter. as your tired to implement standalone library.

How/Where I can see the errors?
